code/stable/nopcm/SRS/NoPCM_SRS.tex

```diff
@@ -264,10 +264,10 @@ \subsubsection{Physical System Description}
 \end{figure}
 \subsubsection{Goal Statements}
 \label{Sec:GoalStmt}
-Given the temperature of the heating coil, initial temperature of the water, and material properties, the goal statement are:
+Given the temperature of the heating coil, the initial temperature of the water, and the material properties, the goal statements are:
 \begin{itemize}
-\item[GS1:]predict the temperature of the water over time
-\item[GS2:]predict the change in heat energy in the water over time
+\item[Predict-Water-Temperature:\phantomsection\label{waterTempGS}]Predict the temperature of the water over time.
```

Comment:
These goals are a subset of the goals for SWHS.  I know it isn't part of this commit, but should we be thinking about ways to reuse these goals?  We have two members of  program family here.  We should find a way to only write their common parts once.  @bmaclach, if this is something that you agree makes sense for the future, can you please create an issue.

Reply:
I actually did re-use the SWHS goals. If you look at the Goals.hs file I added for NoPCM, all it does is import these 2 goals from SWHS and re-export them. Did you have an idea for even further re-use, or is that what you had in mind?

Reply:
Sounds great @bmaclach.  I didn't look too closely at the code.  What you describe is what I had in mind.  If the goal statements change, we only have to change them in one place.  This is another anecdote to keep in mind when writing about the advantages of Drasil.  (@szymczdm, this is an example you might want to highlight in your journal paper.)  :smile: